### DISCUSSION TITLE
Missing stars much brighter than the star catalog limit

### DISCUSSION THREAD

**Discussion by LouisStrous** [Stellarium/stellarium#2088] [General]
I found that (at least) 5 stars with magnitudes between 10.2 and 12.4 are missing from Stellarium's star catalogs in a small (around 0.5 degree) patch of sky near the Pleiades, even though many much fainter stars are included.

I'm using Stellarium 0.21.2 with star catalogs up to and including catalog 7 installed.  This should cover magnitudes up to 15, so I was surprised that the missing stars were missing even though they are much brighter than the combined catalog limit (and I was somewhat disappointed that I hadn't discovered new asteroids or novas...).

I'm including two Stellarium screenshots as evidence.  One shows a screenshot with "Deep Sky objects background images" turned on, and one with that turned off.  I've drawn labelled arrows pointing to the relatively bright stars in the screenshot with the background image that are missing from the screenshot without the background image.

The Stellarium manual (Appendix C) says that catalogs 4 and 5 are based on Tycho data.  According to SIMBAD, 4 of the 5 missing stars have Tycho designations (HD 282974 = TYC 1800-1816-1; HD 282973 = TYC 1800-1683-1; HD 282972 = TYC 1800-1718-1; BD+23 551 = TYC 1800-1852-1).

So there's a puzzle here.  Are those stars not displayed by Stellarium even though they are in the Stellarium star catalogs?  Or are they missing from the Stellarium star catalogs even though they are included in the Tycho catalog?

Regards,
Louis Strous

![stellarium-001-ann](https://user-images.githubusercontent.com/5446337/145074619-768a4634-72a4-473a-9f27-86eada9da667.png)
![stellarium-002](https://user-images.githubusercontent.com/5446337/145074633-9829fd42-5062-4378-88d2-b8fa7a23b30c.png)

**Comment by alex-w** [comment]
This is an artifacts in our star catalog.

**Reply by LouisStrous** [reply]
Is anybody looking into getting newer star catalogs?  If not, then I could give it a try.

**Reply by gzotti** [reply]
You are more than welcome. In the long run we of course want to have GAIA data (necessarily combined with Hipparcos for the brighter stars), and fix shortcomings like
* 3D spatial motion (accurate motion, and see magnitude changes over millennia)
* avoid that faster stars which have moved out of their original "zone" become invisible if the zone is off screen (Arcturus in 70.000 or so?)
* Some users have requested HRD display this also needs more complete data
* other things?
